posthog/admin/admins/redis_admin.py

```diff
@@ -0,0 +1,134 @@
+from django import forms
+from django.contrib import admin, messages
+from django.http import HttpRequest
+from django.utils.translation import ngettext
+
+
+class RedisMutationForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["redis_key"].widget = forms.TextInput(attrs={"size": 50})
+        self.fields["value"].widget = forms.Textarea(attrs={"rows": 10, "cols": 50})
+
+    def save(self, commit=True):
+        self.cleaned_data["status"] = "Created"
```

Comment:
For some reason the form is not using the default value, so I had to add it here.